lib/qb_backend/posts/manual.ex

```diff
@@ -53,4 +55,27 @@ defmodule QbBackend.Posts.Manual do
     |> Manual.changeset(attrs)
     |> put_assoc(:profile, profile)
   end
+@doc """
+  This function takes a bookmark and a manual and creates an association between them
+  """
+  @spec add_to_bookmark(Manual.t(), Bookmark.t()) :: {:ok, Manual.t()} | {:error, Ecto.Changeset.t()}
+  def add_to_bookmark(%Manual{} = manual, %Bookmark{} = bookmark) do
```

Comment:
I am not so sure that we need the functions in the Manual for adding a manual to a bookmark, I think we should only have the ones in the bookmark since the bookmark is the item that `owns` in this relationship

Reply:
Okay.

Reply:
@WanjikuMac  I actually thought a lot about this last evening and I think you should keep these functions but convert the `bookmark - manuals` relationship to many to many since at some point we are going to want to know both the following 
- how many times has this manual been bookmarked
- how many manuals does this bookmark contain 

see

Reply:
@WanjikuMac I thought about this situation and I think I changed my mind, I realized that we may need to keep these functions however we will need to convert the relationship to a `many-to-many` since it is very likely that down the line we will need to answer these two questions 
- How many manuals are in a bookmark 
- In how many bookmarks is a manual

Reply:
@WanjikuMac I thought about this situation and I think I changed my mind, I realized that we may need to keep these functions however we will need to convert the relationship to a `many-to-many` since it is very likely that down the line we will need to answer these two questions 
- How many manuals are in a bookmark 
- In how many bookmarks is a manual

These two would need us to have and maintain a bidirectional relationship I think

Reply:
@WanjikuMac I thought about this situation and I think I changed my mind, I realized that we may need to keep these functions however we will need to convert the relationship to a `many-to-many` since it is very likely that down the line we will need to answer these two questions 
- How many manuals are in a bookmark 
- In how many bookmarks is a manual

These two would need us to have and maintain a bidirectional relationship I think